File: java/fastcampus-java-spring/part2/Chapter2/src/ch12/Person.java
package ch12;

public class Person {
	
	String name;
	int age;
	
	public Person() {
		this("no name", 1);
	}
	
	public Person(String name, int age) {
		this.name = name;
		this.age = age;
	}
	
	public void showPerson() {
		System.out.println(name + "," + age);
	}
	
	public Person getPerson() {
		return this;
	}
	
	public static void main(String[] args) {
		Person person = new Person();
		person.showPerson();
		
		System.out.println(person);
		
		Person person2 = person.getPerson();
		System.out.println(person2);
	}
}

File: flutter/udemy-sangwookcho-flutter-provider/dart_class/bin/person.dart
import 'package:equatable/equatable.dart';

class Person extends Equatable {
  final Stream name;
  final int age;

  Person({
    required this.name,
    required this.age,
  });

  @override
  List<Object?> get props => [name, age];

  @override
  bool get stringify => true;
}
